openedx/core/djangoapps/credit/api.py

```diff
@@ -55,18 +55,21 @@ def set_credit_requirements(course_key, requirements):
                         "name": "i4x://edX/DemoX/edx-reverification-block/assessment_uuid",
                         "display_name": "Assessment 1",
                         "criteria": {},
+                        "order": 1,
                     },
                     {
                         "namespace": "proctored_exam",
                         "name": "i4x://edX/DemoX/proctoring-block/final_uuid",
                         "display_name": "Final Exam",
                         "criteria": {},
+                        "order": 2,
```

Comment:
Nitpick: Since we are getting credit requirements in order of [0, 1, 2, ....] so this line should be`"order": 1` and assessment should get `"order": 2` respectively.